docs/content/preview/secure/enable-authentication/ysql-login-profiles.md

```diff
@@ -0,0 +1,209 @@
+- T (LOCKED(TIMED)): Role is locked for a specified duration.
+
+A role is moved to the LOCKED(TIMED) state when the number of consecutive failed attempts exceeds the limit. The interval (in seconds) to lock the role is read from `pg_yb_profile.prfpasswordlocktime`. The interval is added to the current timestamp and stored in `pg_yb_role_profile.pg_yb_rolprflockeduntil`. Login attempts by the role before `pg_yb_role_profile.pg_yb_rolprflockeduntil` will fail. If the column is NULL, then it is moved to LOCKED state instead.
+
+When the role successfully logs in after `pg_yb_role_profile.pg_yb_rolprflockeduntil`, the role is moved to the OPEN state, and is allowed to log in. Failed attempts after the lock time out period don't modify `pg_yb_role_profile.pg_yb_rolprflockeduntil`. -->
+
+## Manage login profiles
+
+When profiles are enabled, you can manage login profiles using the following commands:
+
+- CREATE PROFILE
+- DROP PROFILE
+- ALTER ROLE
+
+Only superusers can create or drop profiles, and assign profiles to roles.
+
+To create a profile, do the following:
+
+```sql
+CREATE PROFILE myprofile LIMIT
+  FAILED_LOGIN_ATTEMPTS <number>;
+  [PASSWORD_LOCK_TIME <days>];
+```
+
+Note that PASSWORD_LOCK_TIME is optional, and timed locking is not currently implemented.
+
+To drop a profile, do the following:
+
+```sql
+DROP PROFILE myprofile;
+```
+
+To assign a profile to a role, do the following:
+
+```sql
+ALTER ROLE myuser PROFILE myprofile;
+```
+
+To remove a profile from a role, do the following:
+
+```sql
+ALTER ROLE myuser NOPROFILE;
+```
+
+Note that you should remove the association between a role and its profile using `ALTER ROLE ... NOPROFILE` before dropping a role.
+
+### Lock and unlock roles
+
+To unlock a role that has been locked out, do the following:
+
+```sql
+ALTER ROLE myuser ACCOUNT UNLOCK;
+```
+
+To lock a role so that it can't log in, do the following:
+
+```sql
+ALTER ROLE myuser ACCOUNT LOCK;
+```
+
+### View profiles
+
+The `pg_yb_profile` table lists profiles and their attributes.
+
+To view profiles, enter the following command:
+
+```sql
+SELECT * FROM pg_yb_profile;
+```
+
+You should see output similar to the following:
+
+```output
+  prfname  | prfmaxfailedloginattempts | prfpasswordlocktime 
+-----------+---------------------------+---------------------
+ myprofile |                         3 |                   0
+(1 row)
+```
+
+The following table describes the columns and their values:
+
+| COLUMN | TYPE | DESCRIPTION |
+| :---- | :--- | :---------- |
+| `prfname` | name | Name of the profile. Must be unique. |
+| `prfmaxfailedloginattempts` | int | Maximum number of failed attempts allowed. |
+| `prfpasswordlocktime` | int | Interval in seconds to lock the account. NULL implies that the role will be locked indefinitely. |
+
+### View role profile information
+
+The `pg_yb_role_profile` table lists role profiles and their attributes.
+
+To view role profiles, enter the following command:
```

Comment:
To view role profiles, enter the following command:   >   To view role profiles, execute the following statement: